worldwide/walking-cycling.json

```diff
@@ -0,0 +1,95 @@
+{
+  "type": "FeatureCollection",
+  "name": "walking-cycling",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-122.448, 37.772]
+      },
+      "properties": {
+        "place_name": "Panhandle, San Francisco",
+        "zoom": 17,
+        "place_description": "Cycleways and footways.",
+        "tags": ["road:cycleway", "road:footway"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [2.309, 48.869]
+      },
+      "properties": {
+        "place_name": "Champs-Elysees, Paris",
+        "zoom": 17,
+        "place_description": "Polygon pedestrian roads at high zoom.",
+        "tags": ["road:pedestrian"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [12.586, 55.681]
+      },
+      "properties": {
+        "place_name": "Kongens Nytorv, Copenhagen",
+        "zoom": 17,
+        "place_description": "Fences and hedges."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-2.814, 56.349]
+      },
+      "properties": {
+        "place_name": "Old Course at St Andrews",
+        "zoom": 17,
+        "place_description": "Golf course."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-122.924, 50.107]
+      },
+      "properties": {
+        "place_name": "Whistler Blackcomb",
+        "zoom": 14,
+        "place_description": "Pistes, trails, and tracks.",
+        "tags": ["road:piste", "road:trail", "road:track"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [2.3, 48.858]
```

Comment:
Possibly move this point to the intersection.
<img width="334" alt="Screen Shot 2019-12-13 at 2 30 19 PM" src="https://user-images.githubusercontent.com/14204192/70836483-1d889d80-1db5-11ea-9b2e-84b37e814759.png">